Streetcode/Streetcode.XUnitTest/MediatRTests/StreetCode/Streetcode/GetAllStreetcodesHandlerTests.cs

```diff
@@ -64,9 +71,104 @@ public async Task Handle_ReturnsCorrectType()
             // act
 
             var result = await handler.Handle(new GetAllStreetcodesQuery(request), CancellationToken.None);
-            
+
             // assert
             Assert.IsAssignableFrom<GetAllStreetcodesResponseDTO>(result.Value);
         }
+        [Fact]
+        public async Task FindStreetCodeWithMatchTitleTest()
+        {
+            // Arrange
+            var request = new GetAllStreetcodesRequestDTO
+            {
+                Title = "Some Title"
+            };
+
+            var query = new GetAllStreetcodesQuery(request);
+
+            var (handler, repositoryWrapperMock) = SetupMockObjectsAndGetHandler(out var streetcodeRepositoryMock, out var mapperMock);
+
+            IQueryable<StreetcodeContent> streetcodes = new List<StreetcodeContent>().AsQueryable();
+
+            // Act
+            var result = await handler.Handle(query, default);
+
+            // Assert
+            Assert.NotNull(result);
+            Assert.True(result.IsSuccess);
+            Assert.NotNull(result.Value);
+
+            var response = result.Value;
```

Comment:
You do not need this variable. You can use result.Value.Streetcodes like parameter